### PR TITLE
Fix LoadInput type gen

### LINKED ISSUE #916: LoadInput Type generation incorrect
### Describe the bug

If there is both a page and a layout query, the type generation creates 2 separate types for the LoadInput value.

This is due to the houdini-svelte codegen plugin. 

https://github.com/HoudiniGraphql/houdini/blob/f535af8de5838722888de944ca4a21d7450f0b8e/packages/houdini-svelte/src/plugin/codegen/routes/index.ts#L185

### Severity

annoyance

### Steps to Reproduce the Bug

1. Initialize a project
2. Have both a layout load and page load queries with inputs
3. Generated see generated load input incorrect

### Reproduction

_No response_

### PR DESCRIPTION
**Pull request by sjcobb2022**
Fixes #916

### To help everyone out, please make sure your PR does the following:

- [x] Update the first line to point to the ticket that this PR fixes
- [x] Add a message that clearly describes the fix
- [x] If applicable, add a test that would fail without this fix
- [x] Make sure the unit and integration tests pass locally with `pnpm run tests` and `cd integration && pnpm run tests`
- [x] Includes a changeset if your fix affects the user with `pnpm changeset`

This should fix everything!

### CODE REVIEW

**Comment by sjcobb2022** [comment]
There is an error that I missed quickly. let me fix that quickly.

**Comment by sjcobb2022** [comment]
@AlecAivazis Should be good to go now!

File: packages/houdini-svelte/src/plugin/codegen/routes/index.ts [hunk 1]
```diff
@@ -178,8 +178,8 @@ export default async function svelteKitGenerator(
 				// main bulk of the work done here.
 				typeExports = typeExports
 					//we define the loadInput, checks if any queries have imports
-					.concat(append_loadInput(layoutQueries))
-					.concat(append_loadInput(pageQueries))
+					.concat(append_loadInput([...layoutQueries, ...pageQueries]))
+					// .concat(append_loadInput(pageQueries))
```
Comment (AlecAivazis) [inline packages/houdini-svelte/src/plugin/codegen/routes/index.ts:182]: Mind cleaning up this comment?

**Review by AlecAivazis — CHANGES_REQUESTED** [review]
Thanks for getting this together so quickly! One small tweak and i think it's good go

**Review by AlecAivazis — APPROVED** [review]
Thanks!

**Pull request merged**